### PR TITLE
docs: updating FAQs for 2nd and 3rd PR

### LINKED ISSUE #41: Documentation: How to sync the main repo and own repo in GitHub.
To sync the main repo and own repo,

1. pointing to the main repo and pull from it, for that create a new remote in local for pointing the main repo . 
    `git remote add upstream https://github.com/twindle-co/twindle.git`
2. To download all the changes and commits from main Repo.
    `git fetch upstream`
3. The current branch will be updated to reflect the merge.
    `git merge upstream/main`
4. push the changes to own repository.
     `git push origin main`

### PR DESCRIPTION
**Pull request by sammjainn**
closes #41 #211 #163

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -1,80 +1,102 @@
...
 **How to fork?**
-- We'll do a video about it, and we can share the link in this answer.
+
+- This is covered in the video for making first pull request.
 
 **How to create first PR?**
-- Pictures of step by step [here](https://github.com/twindle-co/twindle/issues/57)
-- Video of PR steps:  [![here](http://img.youtube.com/vi/bzaBiQQl6fU/0.jpg)](https://youtu.be/bzaBiQQl6fU)
+
+- Pictures of step by step: [click here](https://github.com/twindle-co/twindle/issues/57)
```
Comment (UnevenCoder) [inline docs/FAQ.md:67]: What do u think of making it 
[Pictures of ... ](url)

File: docs/FAQ.md [hunk 2]
```diff
@@ -1,80 +1,102 @@
...
+- Video of PR steps: [click here](https://youtu.be/bzaBiQQl6fU)
+
+**How to create second PR?**
+
+- Video tutorial: [click here](https://www.youtube.com/watch?v=bPBTumdMhyQ&ab_channel=Twindle)
```
Comment (UnevenCoder) [inline docs/FAQ.md:72]: [Video tutorial](url)

File: docs/FAQ.md [hunk 3]
```diff
@@ -1,80 +1,102 @@
...
 
 **What are the two necessary tasks for being a part of Twindle (through Github)?**
+
 - Create a new file `firstname_lastname.md` inside the `firstpr` directory, commit your change and raise your first Pull Request.
 - Make few changes in the `firstname_lastname.md` file, commit, and use a "CLI" to raise your second Pull Request.
 
-### Logo 🎨
+### Youtube 💻
+
+Find all tutorials and recorded meets on this [channel](https://www.youtube.com/channel/UCKxUmbHq5P5pd5IyUiZ8MHA)
```
Comment (UnevenCoder) [inline docs/FAQ.md:86]: On this [channel] can be #

recorded meets [here](url)

File: docs/FAQ.md [hunk 4]
```diff
@@ -1,80 +1,102 @@
...
 
 **Where to make the logo from?**
+
 - May use free platforms like Canva, Gimp, or Inkspace
 - Use photoshop (if you already have the software)
 
 **How to share the logo I made?**
-- Comment on the issue **[#4 "We need a logo"](https://github.com/twindle-co/twindle/issues/4)** raised by johnjacobkenny
+
+- Comment on the issue **#4 "We need a logo"** raised by johnjacobkenny
```
Comment (UnevenCoder) [inline docs/FAQ.md:97]: You removed the link ?

File: docs/FAQ.md [hunk 5]
```diff
@@ -1,80 +1,102 @@
...
 - Share in the respective twitter group you belong to.
 
 **Is it necessary to make a logo?**
-- No, it is just a part of the project. So, if you are not comfortable with it, its okay.
-
-
 
+- No, it is just a part of the project. So, if you are not comfortable with it, its okay.
```
Comment (UnevenCoder) [inline docs/FAQ.md:102]: it's

**Review by UnevenCoder — CHANGES_REQUESTED** [review]
I requested some changes just to finish most work on it .. 

Pretty good style , great job

**Comment by proful** [comment]
please fix merge conflicts

**Comment by piratelicorne** [comment]
Hi, I added them in the ONBOARDING.md, because yesterday the tutorials were move from FAQ to ONBOARDING -> #306

File: docs/FAQ.md (hunk 4 above)
Comment (sammjainn) [reply docs/FAQ.md:97]: No, I made changes in wiki and copied from there. Someone else must've done this

File: docs/FAQ.md (hunk 5 above)
Comment (sammjainn) [reply docs/FAQ.md:102]: I didn't do this, I'll correct it.

**Pull request closed without merging**